databricks_cli/unity_catalog/delta_sharing_cli.py

```diff
@@ -460,21 +475,29 @@ def get_recipient_cli(api_client, name):
 @eat_exceptions
 @provide_api_client
 def update_recipient_cli(api_client, name, new_name, comment, owner,
-                         allowed_ip_address, json_file, json):
+                         allowed_ip_address, custom_property, json_file, json):
     """
     Update a recipient.
 
     The public specification for the JSON request is in development.
     """
     if ((new_name is not None) or (comment is not None) or (owner is not None) or
-        len(allowed_ip_address) > 0):
+        len(allowed_ip_address) or len(custom_property) > 0):
         if (json_file is not None) or (json is not None):
             raise ValueError('Cannot specify JSON if any other update flags are specified')
         data = {'name': new_name, 'comment': comment, 'owner': owner}
         if len(allowed_ip_address) > 0:
             data['ip_access_list'] = {}
             if len(allowed_ip_address) != 1 or allowed_ip_address[0] != '':
                 data['ip_access_list']['allowed_ip_addresses'] = allowed_ip_address
+        if len(custom_property) > 0:
+            data['properties_kvpairs'] = {}
+            if len(custom_property) != 1 or custom_property[0] != '':
+                parsed_custom_properties = [property_str.split('=')
```

Comment:
Please limit max splits to 2, so the tuple expansion below doesn't break with more (e.g. `x=y=z`).

Reply:
Done.